components/base/kinematicbase/execution.go

```diff
@@ -241,7 +243,6 @@ func (ptgk *ptgBaseKinematics) trajectoryToArcSteps(
 		nextStep.subTraj = append(nextStep.subTraj, trajPt)
 		nextLinVel := r3.Vector{0, trajPt.LinVel * ptgk.linVelocityMMPerSecond, 0}
```

Comment:
A general question to better my understanding. 
We construct some `traj` value which itself has type `[]*TrajNode`.
We then say that `lastLinVel` and `lastAngVel` are related to `traj[0]`.
We then iterate across the `traj` value.
We then say that `nextLinVel` and `nextAngVel` are related to the current `trajPt`.
However, why is there not an issue when `trajPt == traj[0]`?

Reply:
That cannot cause issues for `traj[0]`, because it is guaranteed that for iter 0, `lastAngVel` and `lastLinVel` will always match the next ones, so there will be no new arc step.